app/jobs/bulk_sync_courses_to_find_job.rb

```diff
@@ -0,0 +1,26 @@
+class BulkSyncCoursesToFindJob < ApplicationJob
+  queue_as :find_sync
+
+  def perform
```

Comment:
This is relatively subjective, but I'm not a fan about having non-trivial logic in background jobs. I prefer the job being 1-2 lines long and it calling a method on a model or a service. This allows testing the logic separately from testing the job.

Reply:
moved to recruitment cycle model
 to get the syncable courses